js/index.js

```diff
@@ -1,227 +0,0 @@
-  let sketchCanvas = createCanvas(innerWidth, innerHeight);
-  player = new Player(25, 0, 50, 50);
-  floorPos_y = height - 20;
-  if (!gameStarted) changeStyle("intro");
-  gameStarted = true;
-
-  gameStatus = "";
-  platforms = [
-    new Platform(100, height - 200, 200, 20),
-    new Platform(600, height - 200, 100, 20),
-    new Platform(1000, height - 200, 200, 20),
-    new Platform(1400, height - 200, 50, 20),
-    new Platform(-1, height - 20, 500, 20, "green"),
-    new Platform(800, height - 20, 500, 20, "green"),
-    new Platform(1600, height - 20, 100, 20, "yellow", true),
-  ];
-
-  enemies = [
-    new Enemy(createVector(310, height - 190), 2 * PI),
-    new Enemy(createVector(1000, height - 190), 2 * PI),
-    //new Enemy(createVector(1300, height - 20), PI),
-    
-  ];
-
-  clouds = [
-    { x_pos: 100 - width, y_pos: 150, size: 80 },
-    { x_pos: 500 - width, y_pos: 80, size: 80 },
-    { x_pos: 850 - width, y_pos: 120, size: 80 },
-    { x_pos: 100, y_pos: 150, size: 80 },
-    { x_pos: 500, y_pos: 80, size: 80 },
-    { x_pos: 850, y_pos: 120, size: 80 },
-    { x_pos: 100 + width, y_pos: 150, size: 80 },
-    { x_pos: 500 + width, y_pos: 80, size: 80 },
-    { x_pos: 850 + width, y_pos: 120, size: 80 },
-    { x_pos: 200 + width * 2, y_pos: 80, size: 80 },
-    { x_pos: 550 + width * 2, y_pos: 120, size: 80 },
-    { x_pos: 900 + width * 2, y_pos: 150, size: 80 },
-    { x_pos: 200 + width * 3, y_pos: 80, size: 80 },
-    { x_pos: 500 + width * 3, y_pos: 120, size: 80 },
-    { x_pos: 750 + width * 3, y_pos: 150, size: 80 },
-  ];
-
-  loop();
-}
-
-function draw() {
-  if (gameStarted) {
-    background(color(100, 155, 255));
-    noStroke();
-    textSize(20);
-    fill("white");
-    text(`Lives: ${lives}`, 20, 20);
-
-    // Draw clouds
-    clouds.forEach((cloud) => {
-      drawCloud(cloud);
-      // Movement for clouds, whilst on the visible part of the stage
-      if (cloud.x_pos < 750 + width * 3) {
-        cloud.x_pos += 0.3;
-        // When clouds go off the visible part of the stage, their x_pos is set to the start of the stage
-      } else {
-        cloud.x_pos = 100 - width;
-      }
-    });
-
-    platforms.forEach((platform) => platform.draw());
-
-    if (enemies) {
-      // Draw ball n chains
-      for (let i = 0; i < enemies.length; i++) {
-        enemies[i].draw();
-        enemies[i].updateGC_Pos(player.x, player.y);
-        const isContact = enemies[i].checkContact(player.x, player.y);
-        if (isContact) {
-          if (lives > 1) {
-            lives--;
-            startGame();
-          } else if (lives === 1) {
-            lives--;
-            startGame();
-          }
-        }
-      }
-    }
-
-    player.draw();
-
-    //ScrollWindow
-    if (keys.right.pressed && player.x < 400) {
-      player.velX = 5;
-    } else if (keys.left.pressed && player.x > 100) {
-      player.velX = -5;
-    } else {
-      player.velX = 0;
-      if (keys.right.pressed) {
```

Comment:
this could be refactored. instead of storing a boolean in the right/left pressed and calling things like so, make this a function where you can input an "increment" value which could be -3 or 3 as an argument. 

something like this: 
```js 
function updateThings(amount) {
  platforms.forEach((platform) => (platform.x += amount));
  enemies.forEach((enemy) => (enemy.vector.x += amount));
}

if (left) updateThings(-3)
if (right) updateThings(3)
```